config/testdata/conf.sns-test.yml

```diff
@@ -0,0 +1,16 @@
+route:
```

Comment:
Who is using this file?

Reply:
We'll use it for a config test in config_test.go.